pkg/loki/client.go

```diff
@@ -8,12 +8,14 @@ import (
 )
 
 type LokiClient struct {
-	baseUrl string
+	Client  *http.Client
+	BaseUrl string
```

Comment:
does BaseUrl and client need to be exported?

Reply:
for the tests yes I think so.

Reply:
but I don't see them used in tests?